src/net.h

```diff
@@ -345,6 +344,40 @@ class V1TransportSerializer : public TransportSerializer {
     void prepareForTransport(CSerializedNetMsg& msg, std::vector<unsigned char>& header) const override;
 };
 
+/**
+ * Placeholder for total network traffic. Split by direction, network, connection
+ * type and message type (byte and message counts).
+ */
+class NetStats
+{
+public:
+    struct MsgStat {
+        std::atomic_uint64_t byte_count; //!< Number of bytes transferred.
+        std::atomic_uint64_t msg_count;  //!< Number of messages transferred.
+
+        MsgStat() : byte_count{0}, msg_count{0} {}
```

Comment:
nit: This constructor could use the default keyword since `byte_count` and `msg_count` will default to 0.

e.g. https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#c80-use-default-if-you-have-to-be-explicit-about-using-the-default-semantics

```
 MsgStat() = default;
 ```

Reply:
Ooooo thank you! C++ rookie here :smile: 

Fixed in f5c8b5f78f93bea96e6846ef9d5b787269029b50